vipps-checkout-api-quick-start.md

```diff
@@ -169,7 +165,7 @@ curl https://apitest.vipps.no/checkout/v3/session/UNIQUE-SESSION-REFERENCE \
 -H "client_id: YOUR-CLIENT-ID" \
 -H "client_secret: YOUR-CLIENT-SECRET" \
 -H "Ocp-Apim-Subscription-Key: YOUR-SUBSCRIPTION-KEY" \
--H "Merchant-Serial-Number: 123456" \
+-H "Merchant-Serial-Number: YOUR-MSN" \
 -H "Vipps-System-Name: acme" \
```

Comment:
Maybe instead of acme "YOUR-COMPANY-NAME" or something similar?

Reply:
I updated them now. :-)